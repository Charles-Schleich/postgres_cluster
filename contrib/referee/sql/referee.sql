CREATE EXTENSION referee;

SELECT * FROM referee.decision;

SELECT referee.get_winner(1);
SELECT referee.get_winner(2);
SELECT referee.get_winner(4);
SELECT referee.get_winner(1);
SELECT * FROM referee.decision;

SELECT referee.clean();
SELECT referee.get_winner(4);
SELECT referee.get_winner(2);
SELECT referee.get_winner(1);
SELECT referee.get_winner(4);
SELECT * FROM referee.decision;

SELECT referee.clean();
SELECT referee.clean();
SELECT * FROM referee.decision;
