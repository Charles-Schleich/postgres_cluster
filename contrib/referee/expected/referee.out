CREATE EXTENSION referee;
SELECT * FROM referee.decision;
 key | node_id 
-----+---------
(0 rows)

SELECT referee.get_winner(1);
 get_winner 
------------
          1
(1 row)

SELECT referee.get_winner(2);
 get_winner 
------------
          1
(1 row)

SELECT referee.get_winner(4);
 get_winner 
------------
          1
(1 row)

SELECT referee.get_winner(1);
 get_winner 
------------
          1
(1 row)

SELECT * FROM referee.decision;
  key   | node_id 
--------+---------
 winner |       1
(1 row)

SELECT referee.clean();
 clean 
-------
 t
(1 row)

SELECT referee.get_winner(4);
 get_winner 
------------
          4
(1 row)

SELECT referee.get_winner(2);
 get_winner 
------------
          4
(1 row)

SELECT referee.get_winner(1);
 get_winner 
------------
          4
(1 row)

SELECT referee.get_winner(4);
 get_winner 
------------
          4
(1 row)

SELECT * FROM referee.decision;
  key   | node_id 
--------+---------
 winner |       4
(1 row)

SELECT referee.clean();
 clean 
-------
 t
(1 row)

SELECT referee.clean();
 clean 
-------
 t
(1 row)

SELECT * FROM referee.decision;
 key | node_id 
-----+---------
(0 rows)

